R/marginal_model.R

```diff
@@ -87,11 +87,15 @@ as_epidist_marginal_model.epidist_linelist_data <- function(
   if (n_beyond > 0) {
     cli::cli_inform(c(
       "!" = paste0(
-        "Setting {n_beyond} observation time{?s} beyond ",
-        "{threshold} (={obs_time_threshold}x max delay) to Inf. ",
-        "This improves model efficiency by reducing unique observation times ",
-        "while maintaining model accuracy as these times should have ",
-        "negligible impact."
+        "Setting {n_beyond} relative observation time{?s} beyond ",
+        "{threshold} (= {obs_time_threshold} x max delay) to Inf. ",
+        "This improves model efficiency by reducing the number of unique ",
+        "observation times in the data. The impact on model accuracy should ",
```

Comment:
Again if you can edit this down it would be good as I think this is all just too long. If you want add a link out to the function docs i.e see ? for details and check there to see that it has enough context